views/index.ejs

```diff
@@ -49,6 +49,7 @@
         ></button>
         <div class="search_form__wrapper empty">
           <div class="search_form__return icon-arrow-left"></div>
+          <div class="search_form__return-to-list icon-arrow-left" onclick="window.history.go(-1); return false;"></div>
```

Comment:
There is already an arrow element above ready to do stuff. Why not attaching some behavior to it in JS instead of creating yet another hardcoded button here.

Another problem: if I understand correctly the ticket, using the arrow when navigating on a POIs in a list should go back to the list itself. If you always go back in history, you will reach the previous POI if you have navigated between multiple ones from the same list by clicking on them.
This is a very confusing navigation pattern in this feature :(